mic/utils/fs_related.py

```diff
@@ -1133,9 +1133,14 @@ def cleanup(self):
         else:
             os.unlink(self.device)
 
+DEVICE_LOCKFILE = "/var/lock/__mic_loopdev.lock"
```

Comment:
/run/lock also should be checked, /var/lock is the legacy path.

Reply:
https://github.com/systemd/systemd/commit/5b754353282e3ba3cf9c4ffc50579aff4b7d72db

Reply:
Actually both are legacy paths, so sticking with the one used in the original code.